plenum/server/node.py

```diff
@@ -947,11 +947,14 @@ def on_view_change_complete(self):
         # replicas that have already been ordered on master.
         # Test for this case in plenum/test/view_change/
         # test_no_propagate_request_on_different_last_ordered_before_vc.py
-        if not self.view_changer.propagate_primary:
-            for replica in self.replicas.values():
```

Comment:
As of now, every view change is not a propagate primary, so we can just remove his condition